renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>iExecBlockchainComputing/renovate-config"
```

Comment:
Isn't it `iExecBlockchainComputing/renovate` (https://github.com/iExecBlockchainComputing/renovate)?

Reply:
You're right 😄